include/revng/Pipeline/Global.h

```diff
@@ -0,0 +1,211 @@
+#pragma once
```

Comment:
Consider splitting this file in 3 files, one per class.

Reply:
the first two classes are closely related and do not make sense to split, i put the last one in a stand alone file